client/components/RatingsReviews/ReviewList.jsx

```diff
@@ -16,33 +20,48 @@ export const ReviewList = (props) => {
   // const [loading, setLoading] = useState(true);
   const [reviews, setReviews] = useState(<div>Loading reviews...</div>);
   const [moreReviews, setMoreReviews] = useState('');
-  const [initialized, setInitialized] = useState(false);
+  const [numReviews, setNumReviews] = useState(2);
+  const monthNames = ['January', 'February', 'March', 'April', 'May', 'June',
+    'July', 'August', 'September', 'October', 'November', 'December'
+  ];
   useEffect(() => {
+    let counter = 0;
     getReviews(props.id)
       .then(((data) => {
         setReviews(data.data.results.map((result) => {
-          return (
-            <div className='ReviewList' key={result.review_id}>
-              <div className="IndividualReview">
-                <div className="ReviewText">{result.summary}</div>
-                <div className="jStars">Stars Placeholder</div>
-                <div className="jDate">Date Placeholder</div>
-              </div>
-            </div>
-          );
+          counter++;
+          let date = new Date(result.date);
+          result.date = monthNames[date.getMonth()] + ' ' + date.getDate() + ', ' + date.getFullYear();
+          if (counter <= numReviews ) {
```

Comment:
Instead of checking whether the review is under the numReviews, maybe slice the result of getReviews immediately, before adding it to state with setReviews. This is the beauty of promises, you can chain .then blocks together for really clean, semantic (this, then that) code.

For instance, the first thing I would with the result of getReviews is .then(result => results.data.results.slice(0, numReviews)).then(reviews => reviews.map((review, idx) => <IndividualReview result={review} date={new Date(review.date)} key={idx} />). 

This way, your next then block can immediately be .then(renderedReviews => setReviews(renderedReviews))